### PR TITLE
Replace utils/validate.sh with utils/validate.py, Fix #1097

### LINKED ISSUE #1097: Echo line numbers when ruleset coverage checker fails
If libxml allows us to get at the line number for a given tag.

### PR DESCRIPTION
**Pull request by cschanaj**
Hopefully before #14021

### CODE REVIEW

File: utils/validate.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python2.7
```
Comment (J0WI) [inline utils/validate.py]: Would be great to have Python 3 support for new scripts.

Comment (cschanaj) [reply utils/validate.py]: it seems to me that https://github.com/EFForg/https-everywhere-docker-base does not support Python 3, @Hainish can you confirm this?

**Comment by cschanaj** [comment]
@Hainish this PR increase the speed of `rules-test` and make the error message more readable. would you mind to take a look on it?

**Comment by Hainish** [comment]
@cschanaj I have this on my short-list for review.  Thanks for pinging me on this.

**Comment by cschanaj** [comment]
@Hainish never mind 😄 thank you!!

**Pull request merged**